test/test.js

```diff
@@ -80,6 +81,49 @@ describe('Testing suite for genshin impact gacha', () => {
         }
       }
 
+      expect(hasVenti).to.be.true;
+      done();
+    })
+    it('should give a Venti after pulling an SSR that is not Venti (first Venti pull is also acceptable)', done => {
+      const results = [];
+      const balladVenti = new BalladInGoblets();
+      let hasVenti = false;
+      let hasSSR = false;
+
+      // Infinite loop, we want to keep pulling until we discovered a Venti.
+      while (true) {
+        results.push(balladVenti.roll());
+        
+        // Filter all the results by its five star.
+        // If there are any results, store its name in a new array for easier checking.
+        const filteredResults = results[results.length - 1].filter((item) => item.rating === 5);
+        const names = filteredResults.map((e) => e.name);
+        const areNamesFilled = names.length > 0 ? true : false;
```

Comment:
Drop paranthesis in one parameter arrow functions for the sake of consitency
```
        const filteredResults = roll.filter(item => item.rating === 5);
        const names = filteredResults.map(e => e.name);
        const areNamesFilled = names.length > 0;
```
No ternary needed, expressions using comparison operators always resolve to a boolean